tee-worker/omni-executor/executor-worker/src/main.rs

```diff
@@ -90,8 +89,47 @@ async fn main() -> Result<(), ()> {
 
 	match cli.cmd {
 		Commands::Run(args) => {
-			let args_string = std::env::args().collect::<Vec<String>>().join(" ");
-			info!("Executing: {}", args_string);
+			let env_vars = {
```

Comment:
Can you extact this to a function or create `EnvConfig` struct ?

Reply:
Good idea. It's better to have a struct. Easy for maintenance and sanity check.

Reply:
Done.